dev/reference/performance/statement-summary.md

```diff
@@ -0,0 +1,148 @@
+QUERY_SAMPLE_TEXT: select * from employee where id=3100
+```
+
+Description of each field:
+
+| Column Name          | Description                      |
+|:----------------- |:-------------------------------- |
+| SCHEMA_NAME       | The current schema under which the group of SQL statements are executed|
+| DIGEST            | The digest of the SQL statement                   |
+| DIGEST_TEXT       | Normalized SQL statement              |
+| EXEC_COUNT        | Total execution times of SQL statements of this category   |
+| SUM_LATENCY       | Total execution latency (ns) of SQL statements of this category  |
+| MAX_LATENCY       | Maximum latency (ns) of SQL statements of this category  |
+| MIN_LATENCY       | Minimum latency (ns) of SQL statements of this category|
+| AVG_LATENCY       | Average latency (ns) of SQL statements of this category |
+| SUM_ROWS_AFFECTED | Rows impacted by SQL statements of this category |
+| FIRST_SEEN        | Duration of the first execution for SQL statements of this category|
+| LAST_SEEN         |  Duration of the last execution for SQL statements of this category|
+| QUERY_SAMPLE_TEXT | The original SQL statement where SQL statements of this category firstly appeared |
+
+## Troubleshooting examples
+
+This section shows how to use the statement summary feature to troubleshoot SQL performance issues using two sample questions.
+
+### Could high SQL latency be caused by the server end?
+
+In this example, the client shows slow performance with point queries on the employee table. We can perform a fuzzy query using SQL digest text:
+
+```sql
+SELECT avg_latency, exec_count, query_sample_text
+    FROM performance_schema.events_statements_summary_by_digest
+    WHERE digest_text LIKE ‘select * from employee%’;
+```
+
+As shown in the result below, `avg_latency` of 1ms and 0.3ms are in the normal range. Therefore, we conclude that the server end is not the cause, and continue the troubleshooting with the client or the network.
+
+```
++-------------+------------+------------------------------------------+
+| avg_latency | exec_count | query_sample_text                        |
++-------------+------------+------------------------------------------+
+|     1042040 |          2 | select * from employee where name='eric' |
+|      345053 |          3 | select * from employee where id=3100     |
++-------------+------------+------------------------------------------+
+2 rows in set (0.00 sec)
+```
+
+### Which categories of SQL statements consume the longest total time?
+
+To tune the system, we can find out the top 3 categories of SQL statements with the longest time consumption:
+
+```sql
+SELECT sum_latency, avg_latency, exec_count, query_sample_text
+    FROM performance_schema.events_statements_summary_by_digest
+    ORDER BY sum_latency DESC LIMIT 3;
+```
+
+The result shows that the following three SQL categories consume the longest time in total, which require focus in optimization.
```

Comment:
```suggestion
The result shows that the following three SQL categories consume the longest time in total, which require a focus on optimization.
```

Reply:
"require  a focus" sounds weird to me, although it's a countable noun.